src/batch-client/index.ts

```diff
@@ -934,6 +934,18 @@ export class ZimbraBatchClient {
 			singleRequest: true
 		});
 
+	public getAccountDistributionLists = (attrs: String, ownerOf: number) =>
+		this.jsonRequest({
+			name: 'GetAccountDistributionLists',
+			body: {
+				attrs,
+				ownerOf
+			},
+			namespace: Namespace.Account
+		}).then(res => {
+			return { dls: res?.dl ? res.dl : [] };
+		});
```

Comment:
`}).then(res => ({ dls: res?.dl || [] }));`

Reply:
Done

Reply:
Make it inline statement. as per suggestion it is still missing.

Reply:
@miteshsavani810 sorry I missed that point. I have make it inline statement. Please verify.